server.py

```diff
@@ -8,7 +8,7 @@
 
 Base.metadata.create_all(bind=engine)
 
-app = FastAPI()
+app = FastAPI(openapi_prefix="/api")
```

Comment:
we need this so that the docs appear at all, else the proxy wouldnt load the json. Problem: the demos on the api docs do not factor in this route.

They try to query `localhost:80/*` instead of `localhost:80/api/*`

https://fastapi.tiangolo.com/advanced/behind-a-proxy/ tried using the root path in the docker image entrypoint but that did not work at all

Reply:
I changed the nginx config to the one I use on my testserver, this should make it work correctly so I removed the prefix for now